solutions/src/test/java/ru/sberbank/school/task10/ScalableThreadPoolTest.java

```diff
@@ -0,0 +1,87 @@
+package ru.sberbank.school.task10;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ScalableThreadPoolTest {
+    private ThreadPool threadPool;
+
+    @Before
+    public void init() {
+        threadPool = new ScalableThreadPool(3, 12);
+        threadPool.start();
+    }
+
+    @After
+    public void stop() {
+        threadPool.stopNow();
+    }
+
+    @Test(timeout = 500)
+    public void NormalWork() throws InterruptedException {
+        final AtomicInteger counter = new AtomicInteger(0);
+        for (int i = 0; i < 25; i++) {
+            threadPool.execute(() -> {
+                try {
+                    TimeUnit.MILLISECONDS.sleep(5);
+                } catch (InterruptedException ignore) {
+//                        e.printStackTrace();
+                }
+                synchronized (counter) {
+                    System.out.println("Task: " + counter.incrementAndGet() +
```

Comment:
&#39;+&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a743816117>*